examples/simple-authenticated-api/lib/simple-authenticated-api-stack.ts

```diff
@@ -111,23 +112,47 @@ export class SimpleAuthenticatedApiStack extends cdk.Stack {
           oauth_route: "/oauth/tokens/",
         },
 
-        routes: [
+        lambdaRoutes: [
           {
             name: "route1",
-            paths: ["/1/route1"],
+            path: "/1/route1",
```

Comment:
Changing this from paths -> path to make it match the new url routes.

The url routes can be added by a method as well as props - which highlighted that this should always have been a singular path. We have only ever used a single path here - also telling us that it was wrong.

This is a breaking change though - so  I will put it in the release notes.